src/public/PermitInfo.1.json

```diff
@@ -0,0 +1,1378 @@
+    "type": "eip-2612",
+    "version": "1"
+  },
+  "0x8254e26e453eb5abd29b3c37ac9e8da32e5d3299": {
+    "type": "eip-2612",
+    "version": "1"
+  },
+  "0x86772b1409b61c639eaac9ba0acfbb6e238e5f83": {
+    "type": "eip-2612",
+    "version": "1"
+  },
+  "0x88acdd2a6425c3faae4bc9650fd7e27e0bebb7ab": {
+    "type": "eip-2612",
+    "version": "1"
+  },
+  "0x900db999074d9277c5da2a43f252d74366230da0": {
+    "type": "eip-2612",
+    "version": "1"
+  },
+  "0x92d6c1e31e14520e676a687f0a93788b716beff5": {
+    "type": "eip-2612",
+    "version": "1"
+  },
+  "0x949d48eca67b17269629c7194f4b727d4ef9e5d6": {
+    "type": "eip-2612",
+    "version": "1"
+  },
+  "0x956f47f50a910163d8bf957cf5846d573e7f87ca": {
+    "type": "eip-2612",
+    "version": "1"
+  },
+  "0xa0b86991c6218b36c1d19d4a2e9eb0ce3606eb48": {
+    "type": "eip-2612",
+    "version": "2"
+  },
+  "0xa36fdbbae3c9d55a1d67ee5821d53b50b63a1ab9": {
+    "type": "eip-2612",
+    "version": "1"
+  },
+  "0xac3e018457b222d93114458476f3e3416abbe38f": {
+    "type": "eip-2612",
+    "version": "1"
+  },
+  "0xb3999f658c0391d94a37f7ff328f3fec942bcadc": {
+    "type": "eip-2612",
+    "version": "1"
+  },
+  "0xba5bde662c17e2adff1075610382b9b691296350": {
+    "type": "eip-2612",
+    "version": "1"
+  },
+  "0xbe9895146f7af43049ca1c1ae358b0541ea49704": {
+    "type": "eip-2612",
+    "version": "2"
+  },
+  "0xc18360217d8f7ab5e7c516566761ea12ce7f9d72": {
+    "type": "eip-2612",
+    "version": "1"
+  },
+  "0xc5102fe9359fd9a28f877a67e36b0f050d81a3cc": {
+    "type": "eip-2612",
+    "version": "1"
+  },
+  "0xc7283b66eb1eb5fb86327f08e1b5816b0720212b": {
+    "type": "eip-2612",
+    "version": "1"
+  },
+  "0xcafe001067cdef266afb7eb5a286dcfd277f3de5": {
+    "type": "eip-2612",
+    "version": "1"
+  },
+  "0xcd2828fc4d8e8a0ede91bb38cf64b1a81de65bf6": {
+    "type": "eip-2612",
+    "version": "1"
+  },
+  "0xd38bb40815d2b0c2d2c866e0c72c5728ffc76dd9": {
+    "type": "eip-2612",
+    "version": "1"
+  },
+  "0xd46ba6d942050d489dbd938a2c909a5d5039a161": {
+    "type": "eip-2612",
+    "version": "1"
+  },
+  "0xd9fcd98c322942075a5c3860693e9f4f03aae07b": {
+    "type": "eip-2612",
+    "version": "1"
+  },
+  "0xde30da39c46104798bb5aa3fe8b9e0e1f348163f": {
+    "type": "eip-2612",
+    "version": "1"
+  },
+  "0xdef1ca1fb7fbcdc777520aa7f396b4e015f497ab": {
+    "type": "eip-2612",
+    "version": "1"
+  },
+  "0xedb171c18ce90b633db442f2a6f72874093b49ef": {
+    "type": "eip-2612",
+    "version": "1"
+  },
+  "0xf16e81dce15b08f326220742020379b855b87df9": {
+    "type": "eip-2612",
+    "version": "1"
+  },
+  "0xfb5c6815ca3ac72ce9f5006869ae67f18bf77006": {
+    "type": "eip-2612",
+    "version": "1"
+  },
+  "0xfe459828c90c0ba4bc8b42f5c5d44f316700b430": {
+    "type": "eip-2612",
+    "version": "1"
+  },
+  "0xfe9a29ab92522d14fc65880d817214261d8479ae": {
+    "type": "eip-2612",
+    "version": "1"
+  },
+  "0x0000000000004946c0e9f43f4dee607b0ef1fa1c": false,
```

Comment:
I think we shouldn't event add tokens with no eip2612 support to the list

Reply:
I disagree.
Otherwise, we'll check in the UI whether it's permittable.
If we know already it's not, this saves effort in the UI preventing the check.

Reply:
I had the first gut feeling as Sasha, but then i realised about that. If its not in the list, we don't know if its permitable so we need to check